### DISCUSSION TITLE
System info

### DISCUSSION THREAD

**Discussion by KChanduu** [anse-app/anse#14] [General]
Can we shift system info to right side menu & make is permanent like api key & other details ?

**Comment by yzh990918** [comment]
Sorry, I'm not quite sure what you mean by `system info`.

**Comment by KChanduu** [comment]
![image](https://user-images.githubusercontent.com/127119716/236659740-ec133128-85f0-4260-a941-127ebfa17bf7.png)

**Comment by CNSeniorious000** [comment]
I think @KChanduu means that if there be a way to *copy* the chat settings -- most importantly -- the *system info*, it would be easier than clear all, open system info modal, copy it, new conversation, and finally apply it.

As my personal opinion, I think neither adding a `clone` button beside the `delete` button in the conversation list on the left, nor relocating it to the setting panel may be a perfect way, They may decrease simplicity to a certain extent. But I think it is advisable to add a ***fork*** button in the button group floating when a message got hovered 👀

![image](https://github.com/anse-app/anse/assets/74518716/c72d2dff-49f3-4518-aa95-79d9ee570c99)

I think forking conversation is a universal usage, and on the other hand, @KChanduu's demand is included in it. How does everybody think?

**Reply by yzh990918** [reply]
> I think @KChanduu means that if there be a way to _copy_ the chat settings -- most importantly -- the _system info_, it would be easier than clear all, open system info modal, copy it, new conversation, and finally apply it.
> 
> As my personal opinion, I think neither adding a `clone` button beside the `delete` button in the conversation list on the left, nor relocating it to the setting panel may be a perfect way, They may decrease simplicity to a certain extent. But I think it is advisable to add a _**fork**_ button in the button group floating when a message got hovered 👀
> 
> ![image](https://user-images.githubusercontent.com/74518716/237278123-c72d2dff-49f3-4518-aa95-79d9ee570c99.png)
> 
> I think forking conversation is a universal usage, and on the other hand, @KChanduu's demand is included in it. How does everybody think?

I like your idea.

**Comment by KChanduu** [comment]
Also, it will better, if the copy button appear at the end of the message because view is automatically moving towards the bottom. It will be easy to use it there.
